course-mw/enrolment-actor/src/main/scala/org/sunbird/group/GroupAggregatesActor.scala

```diff
@@ -0,0 +1,144 @@
+package org.sunbird.group
+
+import org.apache.commons.collections.CollectionUtils
+import org.apache.commons.collections.MapUtils
+import org.apache.commons.lang3.StringUtils
+import org.sunbird.common.exception.ProjectCommonException
+import org.sunbird.common.models.response.Response
+import org.sunbird.common.models.util.{ProjectLogger}
+import org.sunbird.common.request.Request
+import org.sunbird.common.responsecode.ResponseCode
+import org.sunbird.keys.SunbirdKey
+import org.sunbird.learner.actors.group.dao.impl.GroupDaoImpl
+import org.sunbird.actor.base.BaseActor
+import org.sunbird.cache.CacheFactory
+import org.sunbird.cache.interfaces.Cache
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+class GroupAggregatesActor extends BaseActor {
+
+
+  private val GROUP_MEMBERS_METADATA: java.util.List[String] = java.util.Arrays.asList("name", "userId", "role", "status", "createdBy")
+
+
+  var groupDao: GroupDaoImpl = new GroupDaoImpl()
+  var groupAggregatesUtil: GroupAggregatesUtil = new GroupAggregatesUtil()
```

Comment:
Why do you need to create a util class as an instance? and two why is declared as var whereas it should be val